README.md

```diff
@@ -271,7 +271,7 @@ Alternatively, you can perform a one-time scan on a specific pull request using
 4.b.2. Execute the single scan with the following command:
 
    ```bash
-   java -jar -Dgithub.token=<my-token> lpvs-*.jar --github.pull.request=<PR URL>
+   java -jar -Dgithub.token=<my-token> lpvs-*.jar --github.pull.request=<PR URL> --build.html.report=<Path to report folder>
```

Comment:
Do we need to specify Path to report folder or Path + Filename? 
It's not clear from the description. Pease add more words into Readme.